open-apparel-registry/open-apparel-registry
# Add contributor embed level field

## Overview

Determines what features are available to the contributor when creating an embedded map.

Connects #1335 

## Demo

*Embed level 1*
![Screen Shot 2021-05-24 at 3 28 27 PM](https://user-images.githubusercontent.com/1042475/119397828-b1e27980-bca4-11eb-9376-1d0aad12af3c.png)

*Embed level 2*
![Screen Shot 2021-05-24 at 3 27 16 PM](https://user-images.githubusercontent.com/1042475/119397707-88295280-bca4-11eb-8091-2d9c76b3d9c8.png)

*Embed level 3*
![Screen Shot 2021-05-24 at 3 27 51 PM](https://user-images.githubusercontent.com/1042475/119397772-9c6d4f80-bca4-11eb-8f00-3608c5fe5348.png)

## Testing Instructions

- Go to the Django admin, and navigate to the contributor settings for the logged in user.
- Note the "Embed Level" settings. It should be set to 1. You'll need to change this to test out the embed settings.
- In a separate tab visit the Settings > Embed page. 
- Verify the embed settings follow these rules:
  - Custom fields settings are hidden on level 1 and 2
  - Color settings are limited to two choices on level 1. For level 2 and 3, the full color selector should be displayed.
  - Font settings are hidden on levels 1 and 2

## Checklist
